wagtailio/static/sass/components/_sign-up-form.scss

```diff
@@ -25,19 +25,22 @@
         @include z-index(under);
         position: relative;
         width: auto;
-        height: 50px;
         margin-left: auto;
         display: none;
 
-        @include media-query(large) {
+        @include media-query(medium) {
             display: block;
+            height: 50px;
         }
 
         > svg {
             position: relative;
-            inset: -100px 0 auto auto;
             width: 170px;
             height: 170px;
+
+            @include media-query(medium) {
+                inset: -100px 0 auto auto;
```

Comment:
Nickpick: This is a huge improvement over the current styling, however it's still possble to have the text overlap at some viewports, which given the bright colour of the icon here makes the text really difficuly to read.

<details><summary>Example</summary>

<img width="637" alt="Screenshot 2024-02-06 at 17 39 07" src="https://github.com/wagtail/wagtail.org/assets/35934884/ee71b213-c690-417e-b749-e727f157d645">

</details>

If you altered the DOM a little, you could then constrain the text to a division element and have both the text and icon be responsive to their respective containers, using `display: flex`.


<details><summary>Example using flex</summary>
You could then use `flex-direction: column` to move the icon below the form at mobile. We'd also have to wouldn't have to offset the image using CSS which can be tricky to manage. 

<img width="1197" alt="Screenshot 2024-02-06 at 17 44 03" src="https://github.com/wagtail/wagtail.org/assets/35934884/c2b87f29-4939-4f19-a92e-c1a48712759b">

</details>